x-pack/plugins/security_solution/server/lib/detection_engine/routes/index/get_signals_template.ts

```diff
@@ -8,7 +8,7 @@
 import signalsMapping from './signals_mapping.json';
 import ecsMapping from './ecs_mapping.json';
 
-export const SIGNALS_TEMPLATE_VERSION = 14;
+export const SIGNALS_TEMPLATE_VERSION = 24;
```

Comment:
How does the template incrementing work again? I forget. Maybe we can add a comment describing how / why we update these numbers.

Reply:
I'm not really sure, either 😆 I'll bring it up to the team and add a comment here.

Reply:
Updated in https://github.com/elastic/kibana/pull/89899/commits/a96c773977fbbb5780cb14c2d7cb2f5b89e39fa8